domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -760,8 +792,23 @@ constructor(
     }
 
     override suspend fun setLowLightBoost(lowLightBoost: LowLightBoost) {
-        currentSettings.update { old ->
-            old?.copy(lowLightBoost = lowLightBoost)
+        when (lowLightBoost) {
+            LowLightBoost.DISABLED -> {
+                currentSettings.update { old ->
+                    old?.copy(
+                        lowLightBoost = lowLightBoost,
+                        captureMode = CaptureMode.MULTI_STREAM,
```

Comment:
What if the `CaptureMode` was `SINGLE_STREAM` before disabling low-light boost?

If we can only support LLB in `SINGLE_STREAM` mode, then we should do something similar to what we did with `CameraConstraints.supportedImageFormatsMap` where we have a `Map<CaptureMode, Boolean>`. We can then determine in the UI whether we should allow users to enable LLB depending on which capture mode is enabled (otherwise we disable the button).